src/libsyncengine/update_detection/file_system_observer/computefsoperationworker.cpp

```diff
@@ -38,6 +40,8 @@ void ComputeFSOperationWorker::execute() {
     LOG_SYNCPAL_DEBUG(_logger, "Worker started: name=" << name().c_str());
     auto start = std::chrono::steady_clock::now();
 
+    //_syncPal->_localFSObserverWorker->invalidateSnapshot();
```

Comment:
To re-instate?

Reply:
No sorry, it was just for testing. I removed it.